DTS-STN/Scrum-Poker
# Fix Card 1 shadow and cards outline

### Description

Small PR that fixes a couple issues with the card component. The SVG for Card 1 for whatever reason had an inner drop shadow when the other cards didn't so I removed that. When switching to the 2 column layout, all the cards had an outline around them which looked really ugly, so this fixes that as well.

### What to test for/How to test

1. Pull in branch
2. Type `npm run dev`
3. Create a room
4. Look at cards to ensure the Card 1 looks the same as every other card and that there isn't an outline around every card by default

### Additional Notes

I noticed while reviewing #88 that if you create a room locally and then refresh the page, you get this console warning:

```
next-dev.js?3515:25 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    at Image (webpack-internal:///./node_modules/next/dist/client/image.js:87:22)
    at div
    at Card (webpack-internal:///./components/Card.js:18:119)
```
This obviously has nothing to do with either this or that PR, but something to consider at least. I haven't been able to reproduce this on main so it could be a non-issue.
